addons/website/static/src/js/content/menu.js

```diff
@@ -615,22 +628,14 @@ publicWidget.registry.hoverableDropdown = animations.Animation.extend({
     _onMouseEnter: function (ev) {
         // The user must click on the dropdown if he is on mobile (no way to
         // hover) or if the dropdown is the extra menu ('+').
-        if (config.device.size_class <= config.device.SIZES.SM ||
-            ev.currentTarget.classList.contains('o_extra_menu_items')) {
-            return;
-        }
-        Dropdown.getOrCreateInstance(ev.currentTarget.querySelector('.dropdown-toggle')).show();
+        this._toggleDropdown(ev);
```

Comment:
Not a fan of calling "toggle" when what you really want to do is "show" (or am I misunderstanding something?). But it's a good idea to make a function out of that part, maybe add a second argument to the function to say if we want to show or hide?